### DISCUSSION TITLE
Resizing Broadcast

### DISCUSSION THREAD

**Discussion by PeadarHarvey** [epoptes/epoptes#206] [General]
Hi, first of all thanks for a great piece of software. I am using it in a classroom teaching Computer Science. 

I wondering if t a few things are possible....

Firstly, when I broadcast my screen. My two screens are broadcast. Is it possible that only one screen would be broadcast? 

Secondly, on the client PC's when I broadcast my screen.... and the students resize the window they see very little of the screen. Is it possible that they can see my whole screen just in a smaller window?

**Comment by alkisg** [comment]
Hi @PeadarHarvey, no, you cannot broadcast a single screen.
For the students to see all your screen when you have a bigger resolution, you'll need the RealVNC client, read https://epoptes.org/documentation/vnc/.

**Comment by PeadarHarvey** [comment]
Hi Alkisg, thanks for pointing this out to me. I will try this :-)
